examples/Example1_Fit_Halo_Mass_to_Shear_Catalog.ipynb

```diff
@@ -551,10 +551,10 @@
     "logm_0 = random.uniform(13., 17., 1)[0]\n",
     "\n",
     "#logm_est1 = samplers['minimize'](nfw_to_shear_profile, logm_0,args=[r1, gt_profile1, z1])\n",
```

Comment:
@hsinfan1996 - could you also modify the commented-out line using minimize from

logm_est1 = samplers['minimize'](nfw_to_shear_profile, logm_0,args=[r1, gt_profile1, z1])

to logm_est1 = samplers['minimize'](nfw_to_shear_profile, logm_0,args=[r1, gt_profile1, z1])[0]

At the moment, using the minimize lines as they are return a shape error later on.

Reply:
got it.